tools/src/main.rs

```diff
@@ -0,0 +1,55 @@
+use dialoguer::{theme::ColorfulTheme, Select};
+use std::process::Command;
+use std::process;
+
+fn main() {
+    let cmds = [
+        ("Build Application Services", "cargo build"),
+
+        ("Run all tests for a pull request", "sh ./automation/all_tests.sh"),
+        ("Run all Rust tests", "sh ./automation/all_rust_tests.sh"),
+
+        ("Verify Desktop Environment", "sh ./libs/verify-desktop-environment.sh"),
+        ("Verify Android Environment", "sh ./libs/verify-android-environment.sh"),
+        ("Verify iOS Environment", "sh ./libs/verify-ios-environment.sh"),
+
+        ("Smoke test Android Components", "python3 ./automation/smoke-test-android-components.py"),
+        ("Smoke test Fenix Components", "python3 ./automation/smoke-test-fenix.py"),
+        ("Smoke test Firefox iOS Components", "python3 ./automation/smoke-test-firefox-ios.py"),
```

Comment:
@rfk is this missing any scripts that were introduced as part of the recent work?